analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentPlugin.java

```diff
@@ -1,23 +1,20 @@
 package org.jboss.gm.analyzer.alignment;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.stream.Collectors;
-
 import org.gradle.api.Plugin;
-import org.gradle.api.Project;
-import org.jboss.gm.common.alignment.AlignmentModel;
+import org.jboss.gm.common.alignment.AlignedProject;
 import org.jboss.gm.common.alignment.AlignmentUtils;
+import org.jboss.gm.common.alignment.Module;
 
 /**
  * Results in adding a task with name {@value org.jboss.gm.analyzer.alignment.AlignmentTask#NAME}.
- * It also creates the basic alignment.json file that {@value org.jboss.gm.analyzer.alignment.AlignmentTask#NAME} of each project adds to
+ * It also creates the basic alignment.json file that {@value org.jboss.gm.analyzer.alignment.AlignmentTask#NAME} of each
+ * project adds to
  *
  */
-public class AlignmentPlugin implements Plugin<Project> {
+public class AlignmentPlugin implements Plugin<org.gradle.api.Project> {
```

Comment:
Can we get rid of the package name here an in some of the other places it appears?

Reply:
Yes, that's what I'm actually fixing right now :)

Reply:
:+1: